code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_abilities.dm

```diff
@@ -338,12 +339,12 @@
 	if (WH.curr_effect_type == WARDEN_HEAL_HP)
 		action_icon_result = "warden_rejuvenate"
 		WH.curr_effect_type = WARDEN_HEAL_DEBUFFS
-		to_chat(X, SPAN_XENOWARNING("You will now protect your allies by rejuvenating them!"))
+		to_chat(X, SPAN_XENOWARNING("You will now assist your allies by removing their debuffs!"))
```

Comment:
Do not break the wall . Chata need to stay "ingame" would recomend calling it different rather than debuffs

Reply:
Yessir, I will fix that.